access.asd

```diff
@@ -28,9 +22,4 @@
                         :serial t
                         :components ((:file "access")
                                      (:file "arg-list-manipulation"))))
-  :depends-on (:access :lisp-unit2))
-
-(defmethod asdf:perform ((o asdf:test-op) (c (eql (find-system :access))))
```

Comment:
It hardly matters to me, as I don't anticipate ever returning to full time common lisp work, but this was in there to ensure the tests ran when someone called test. 

Is this no longer needed to run the tests when someone calls `(asdf:test :access)` ?   (or whatever the syntax is?)

Reply:
Just for your information, this method definition is replaced by 

```
:in-order-to ((test-op (load-op "access/test")))
:perform (test-op (op c) (symbol-call '#:access-test '#:run-all-tests)))
```

Good luck with your non-Lisp work!

Reply:
Thanks for your explanation!  I actually saw that in one of your other commits but missed it here.  

Thank you!  Running CL for a decade was very formative for me, and I was sad to move on, but payments and progress demanded it :)